lib/signable/query/client.rb

```diff
@@ -29,7 +29,7 @@ def create(entry_point, object)
         Response.new response
       end
 
-      def delete(entry_poind, id)
+      def delete(entry_point, id)
```

Comment:
This is a bug currently in the repo that we found and fixed.  I'm assuming we don't delete Signable stuff anywhere in our `api` codebase, as if we tried to, via this gem, it wouldn't work 😕